main.py

```diff
@@ -283,6 +283,10 @@ def log_bad_request(request):
     except AttributeError:
         logging.debug("Bad request:{0}".format(str(request)))
 
+@app.route("/test")
```

Comment:
Please do - it's ok for testing locally but there's no reason to have it in the website.
Also please see my other comments.